frontend/components/TableContainer/TableContainer.tsx

```diff
@@ -34,7 +34,7 @@ interface IRowProps extends Row {
   };
 }
 
-interface ITableContainerProps {
+interface ITableContainerProps<T = any> {
```

Comment:
Can this be typed as `ITableContainerProps<R = IRowProps>`?

Reply:
I need to mess with this a bit more. That's similar to what I was going for but couldn't figure out the correct typing to get the TS compiler to be happy. I ideally want to have the types set so we can use TableContainer like this. `<TableContainer<IMdmSolution> ... />` instead of `<TableContainer<IRowProps> ... />`. I'll be looking at improving the typing mentioned here.